### PR TITLE
Lofreq patch

### PR DESCRIPTION
**Pull request by rdmorin**
# Pull Request Checklists
## Checklist for Updated Module
See Changelog.

### CODE REVIEW

File: modules/lofreq/1.0/lofreq.smk [hunk 1]
```diff
@@ -129,10 +133,13 @@ rule _lofreq_filter_vcf:
         CFG["conda_envs"]["bcftools"]
     shell:
         op.as_one_line("""
-        bash {input.lofreq_filter} {input.vcf_all} | bgzip > {output.vcf_all_clean}
+        PATH={SCRIPT_PATH}:$PATH;
+        SCRIPT=$(which lofreq_filter.sh); 
+        echo $SCRIPT;
```
Comment (lkhilton) [inline modules/lofreq/1.0/lofreq.smk]: Can you make this more informative? E.g. `echo "Using the script at $SCRIPT to filter lofreq calls"`

File: modules/lofreq/1.0/lofreq.smk [hunk 2]
```diff
@@ -73,6 +75,9 @@ rule _lofreq_run:
         **CFG["resources"]["lofreq"]
     shell:
         op.as_one_line("""
+        PATH={SCRIPT_PATH}:$PATH;
+        SCRIPT=$(which lofreq2_call_pparallel.py);
+        echo $SCRIPT;
```
Comment (lkhilton) [inline modules/lofreq/1.0/lofreq.smk]: Same as my comment below on line 138 - make a user-readable output.

File: modules/lofreq/1.0/lofreq.smk [hunk 3]
```diff
@@ -73,6 +75,9 @@ rule _lofreq_run:
         **CFG["resources"]["lofreq"]
     shell:
         op.as_one_line("""
+        PATH={SCRIPT_PATH}:$PATH;
+        SCRIPT=$(which lofreq2_call_pparallel.py);
```
Comment (lkhilton) [inline modules/lofreq/1.0/lofreq.smk]: I wonder if using the path to the script instead of `which` here would help ensure it's not inadvertently using the Conda-installed version. Then you could include something like `if [[ $SCRIPT == "" ]]; then echo "Lofreq script not found in module directory. Please ensure you are using the latest version from Master.; fi`

File: modules/lofreq/1.0/src/lofreq_filter.sh [hunk 4]
```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/usr/bin/env bash
```
Comment (lkhilton) [inline modules/lofreq/1.0/src/lofreq_filter.sh:1]: What's the rationale for this change? I always use `#!/bin/bash` in my scripts, curious what the difference is.

**Review by lkhilton — CHANGES_REQUESTED** [review]

File: modules/lofreq/1.0/lofreq.smk (hunk 3 above)
Comment (rdmorin) [reply modules/lofreq/1.0/lofreq.smk]: I'm not sure I understand. How would I use the path to the script? Wouldn't this check only work if the user is already using the latest version?

File: modules/lofreq/1.0/lofreq.smk (hunk 2 above)
Comment (rdmorin) [reply modules/lofreq/1.0/lofreq.smk]: Good call. Done

File: modules/lofreq/1.0/lofreq.smk (hunk 1 above)
Comment (rdmorin) [reply modules/lofreq/1.0/lofreq.smk]: Done but not tested. Do you happen to know if quotes necessary?

Comment (lkhilton) [reply modules/lofreq/1.0/lofreq.smk]: I would use quotes to be safe, yes, to make sure each word in the echo phrase isn't parsed as a separate argument.

File: modules/lofreq/1.0/lofreq.smk (hunk 3 above)
Comment (rdmorin) [reply modules/lofreq/1.0/lofreq.smk]: I understand now. That would indeed protect against any unexpected environment issues. In theory prepending to PATH should make this check unnecessary but from recent experience, it's better not to assume this will always work.

**Comment by rdmorin** [comment]
I believe this is ready for re-review, @lkhilton

**Review by lkhilton — APPROVED** [review]
